rules.d/90-iocost.rules

```diff
@@ -0,0 +1,18 @@
+#  SPDX-License-Identifier: LGPL-2.1-or-later
+#
+#  This file is part of systemd.
+#
+#  systemd is free software; you can redistribute it and/or modify it
+#  under the terms of the GNU Lesser General Public License as published by
+#  the Free Software Foundation; either version 2.1 of the License, or
+#  (at your option) any later version.
+
+SUBSYSTEM!="block", GOTO="iocost_end"
```

Comment:
there should probably also be a DEVTYPE match here, that filters out partition block devices, no?

Reply:
Right, we should filter out partition as well to avoid duplicating work(iocost apply triggered for each partition detection right now). I will update rules. Thanks

Reply:
Filtering out of partitions is added in udev rules in latest push.  Thanks for pointing that.